data/presets/natural/rock.json

```diff
@@ -1,7 +1,14 @@
 {
     "icon": "temaki-boulder2",
     "fields": [
-        "name"
+        "name",
+        "height",
+        "geological_outcrop"
```

Comment:
Something that irritates me a bit is that the proposed added field for `geological=outcrop` to the `natural=stone` preset currently only has very few usages (only 30 map features have this tag combination). The [wiki](https://wiki.openstreetmap.org/wiki/Tag%3Ageological%3Doutcrop) [also](https://wiki.openstreetmap.org/wiki/Tag%3Anatural=stone) doesn't mention this particular usage.